spec/views/content/shared/benefits.html.erb_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+describe "_benefits.html.erb" do
+  let(:subject_name) { "chemistry" }
+  let(:salary_value) { YAML.load_file("config/values/salaries.yml").dig("salaries", "starting", "minshortened") }
+
+  subject do
+    assign(:front_matter, { "subject" => subject_name })
+    assign(:salaries_starting_minshortened, "£31k")
```

Comment:
Is this `£31k` just an example amount? We wouldn't have to update it as salaries changed?

Reply:
it's just an example :)